src/containers/Drawer.tsx

```diff
@@ -1,30 +1,36 @@
-import App, { REFETCH_ON_MOUNT, REFETCH_ON_SUCCESS, HLS_DISPLAY, SERVER_STATE, refetchAtom } from "./App";
-import {
-  DEFAULT_HOST,
-  DEFAULT_PATH,
-  DEFAULT_PROTOCOL,
-  DEFAULT_TOKEN,
-  useServerSdk,
-} from "../components/ServerSdkContext";
-import { useApi } from "./Api";
+import { REFETCH_ON_MOUNT, REFETCH_ON_SUCCESS, HLS_DISPLAY } from "./App";
 import { ThemeSelector } from "../components/ThemeSelector";
 import { GiHamburgerMenu } from "react-icons/gi";
-import { LogSelector, PersistentInput, PersistentExtras } from "../components/LogSelector";
-import { useAtom } from "jotai";
+import { LogSelector, PersistentInput, PersistentExtras, extraSelectorAtom } from "../components/LogSelector";
+import { JellyfishServer, ServerProps } from "./JellyfishServer";
+import { useState } from "react";
+import { CloseButton } from "../components/CloseButton";
+import { atom, useAtom } from "jotai";
+import HLSPlayback from "../components/HLSPlayback";
 
+export const LOCAL_STORAGE_HOST_KEY = "host";
+export const LOCAL_STORAGE_PROTOCOL_KEY = "signaling-protocol";
+export const LOCAL_STORAGE_PATH_KEY = "signaling-path";
+export const DEFAULT_HOST = "localhost:5002";
+export const DEFAULT_PROTOCOL = "ws";
+export const DEFAULT_PATH = "/socket/peer/websocket";
+export const DEFAULT_TOKEN = "development";
+
+export const hostAtom = atom(DEFAULT_HOST);
+export const protocolAtom = atom(DEFAULT_PROTOCOL);
+export const pathAtom = atom(DEFAULT_PATH);
+export const serverTokenAtom = atom(DEFAULT_TOKEN);
+export const serversAtom = atom<Map<string, ServerProps>>(new Map());
```

Comment:
In my opinion, this could be a simple list. I don't see any advantages in using Map. It would simplify iterations and additions

Reply:
I'm using Map to ensure that each server can appear at most once (it still has some drawbacks, for example localhost:port and direct_ip:port can still connect to the same server). It's also easier to remove random element from the map, but surely I can use the List here

Reply:
I'm convinced that map makes it much easier to refetch proper room each time the room is added. I'll leave it as it is for now.